### PR TITLE
WordPress 5.7 and jQuery 3.5.X Compatibility

### PR DESCRIPTION
**Pull request by bordoni**
🎟️ [ET-992]
---
Do not merge this it will likely be in-branch QAed.

[ET-992]: https://theeventscalendar.atlassian.net/browse/ET-992

### CODE REVIEW

**Review by sc0ttkclark — DISMISSED** [review]

File: src/resources/js/v2/rsvp-ari.js [hunk 1]
```diff
@@ -79,20 +79,20 @@ tribe.tickets.rsvp.ari = {};
...
 		// Set the classes for inactive.
 		$guestListButtons.addClass( obj.selectors.guestListItemButtonInactive.className() );
-		$guestListButtons.attr( 'aria-selected', false );
+		$guestListButtons.prop( 'aria-selected', false );
 
 		// Set the active class for the current.
 		const $targetGuestButton = $container.find( obj.selectors.guestListItemButton + '[data-guest-number="' + guestNumber + '"]' );
 		$targetGuestButton.removeClass( obj.selectors.guestListItemButtonInactive.className() );
-		$targetGuestButton.attr( 'aria-selected', true );
+		$targetGuestButton.prop( 'aria-selected', true );
```
Comment (paulmskim) [inline src/resources/js/v2/rsvp-ari.js:95]: these should be `.attr( 'aria-selected', 'true' )` or `attr( 'aria-selected', 'false' )`, the value should be a string of `true` or `false`.

File: src/resources/js/tickets.js [hunk 2]
```diff
@@ -12,767 +15,770 @@ tribe.tickets.editor = {};
...
 
-			var $eventTickets = $( '#event_tickets' );
+		// trigger an event before swapping the panel
+		$eventTickets.trigger( 'before_panel_swap.tickets', { panel: $panel } );
 
-			// trigger an event before swapping the panel
-			$eventTickets.trigger( 'before_panel_swap.tickets', {panel: $panel} );
+		// First, hide them all!
+		$tribe_tickets.find( '.ticket_panel' ).each( function() {
+			$( this ).prop( 'aria-hidden', true );
```
Comment (paulmskim) [inline src/resources/js/tickets.js:213]: this should be `.attr( 'aria-hidden', 'true' )`

File: src/resources/js/tickets.js [hunk 3]
```diff
@@ -12,767 +15,770 @@ tribe.tickets.editor = {};
...
+		} );
 
-			// First, hide them all!
-			$tribe_tickets.find( '.ticket_panel' ).each( function() {
-				$( this ).attr( 'aria-hidden', true );
-			} );
+		// then show the one we want
+		$panel.prop( 'aria-hidden', false );
```
Comment (paulmskim) [inline src/resources/js/tickets.js:217]: this should be `.attr( 'aria-hidden', 'false' )`

File: src/resources/js/v2/rsvp-ari.js (hunk 1 above)
Comment (sc0ttkclark) [reply src/resources/js/v2/rsvp-ari.js:95]: @bordoni Does this still need to be addressed?

**Review by sc0ttkclark — COMMENTED** [review]
Q about something Paul was asking about

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**